src/env.cc

```diff
@@ -250,6 +255,13 @@ Environment::~Environment() {
   // CleanupHandles() should have removed all of them.
   CHECK(file_handle_read_wrap_freelist_.empty());
 
+  // dispose the Persistent references to the compileFunction
+  // wrappers used in the dynamic import callback
+  for (auto & entry : compile_fn_entries) {
+    id_to_function_map[entry->id].Reset();
```

Comment:
This line shouldn’t be needed, we destroy `id_to_function_map` at the end of the destructor and the `node::Persistent`s with it

Reply:
Good point.